doc/userguide/performance/high-performance-config.rst

```diff
@@ -1,19 +1,310 @@
 High Performance Configuration
 ==============================
 
-If you have enough RAM, consider the following options in suricata.yaml to off-load as much work from the CPU's as possible:
+NIC
+---
+
+One of the major dependencies for Suricata's performance is the Network Interface Card. There are many vendors and possibilities. Some NICs have and require their own specific instructions and tools of how to set up the NIC. That way ensuring the greatest benefit when running Suricata. Vendors like Napatech, Netronome, Accolade, Myricom include those tools and documentation as part of their sources.
```

Comment:
I don't see https://github.com/OISF/suricata/pull/4182#discussion_r322269332 being addressed in this PR. Did you forget a commit?

Reply:
I thought i fixed it... will redo